android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/TranslationsState.kt

```diff
@@ -19,6 +20,7 @@ import mozilla.components.concept.engine.translate.TranslationSupport
  * @property isTranslated The page is currently translated.
  * @property isTranslateProcessing The page is currently attempting a translation.
  * @property isRestoreProcessing The page is currently attempting a restoration.
+ * @property translationSize The download size if a given to/from pair is translated.
```

Comment:
```suggestion
 * @property translationDownloadSize The download size if a given to/from pair is translated.
```
I think one common point that is coming up is about describing what the size represents? Just out of curiosity, this should represent the translation engine size of the translation pair?

Reply:
Thanks for taking a look!

Had trouble naming, yes, I think `translationDownloadSize` is more descriptive. 

>I think one common point that is coming up is about describing what the size represents? 

It is the download size of the specific model or models necessary to complete a translation on a given pair. So, for example, `fr` -> `de` will need to download a few ML model files and then run the models on the device to complete the translation. (It'll download `fr` -> `en` and `en` -> `de`, at present.)

Once the user has the models on the device once, no other downloads are needed. For example, "es" -> "en" the first time will be X MB (required download), the next "es" -> "en" request will be 0 MB (no download needed). It would also be 0 MB, if the user had also downloaded the Spanish language "package".

The full "package" or "suite" of models that this [screen downloads](https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD/Full-Page-Translation?type=design&node-id=7341-209489&mode=design&t=tiXPvZUQOJeZojlm-0) is conceptually made up of all the pairs for that language. 

Great call-out, I'll update the comments all around to try and make that more clear!

Reply:
Added some more description here and in the object